### DISCUSSION TITLE
Allow setting custom fields on issue creation

### DISCUSSION THREAD

**Discussion by shyim** [ankitpokhrel/jira-cli#279] [Ideas] [answered]
Hey,

we have stupid custom fields like Team, which needs to be filled on any issue creation. It would be great if we could create a yml file with default values 🤔

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @shyim, there is already an open request for this: #222

I will hopefully work on this as soon as I get some time.

**Reply by shyim** [reply]
I saw it too. idk how to delete this xD
